python/requirements.in

```diff
@@ -1,13 +1,15 @@
+--extra-index-url https://nasa-gcn.github.io/healpy-prerelease-pypi-index/simple/
 architect-functions
 astropy
 astropy_healpix
 cachetools
 email-validator
 fastapi
-healpy
+healpy==0.1.dev1+g500357b
 mangum
 python-multipart  # Required for file uploads in FastAPI; see https://fastapi.tiangolo.com/tutorial/request-files/
 requests
+scipy
```

Comment:
As #2054 removes the scipy dependency, this will add it back. Suggest merging #2054 first, rebasing, and removing scipy from `requirements.in`.